packages/gravity-ui-nunjucks/src/_patterns/00-particles/00-color/02-color-schemes.njk

```diff
@@ -50,10 +50,10 @@
 </table>
 {% endmacro -%}
 
-{% for schemeName, schemeTableData in colorSchemes %}
-<section class="sg-color-scheme grav-u-color-scheme-{{ schemeName }}">
-  <h4>{{ schemeName | title }} color scheme</h4>
-  <p>Can be applied to containers using the <code>grav-u-color-scheme-{{ schemeName }}</code> utility class.</p>
-  {{ colorSchemeTable(schemeTableData) | safe }}
+{% for schemeKey, schemeData in colorSchemes %}
```

Comment:
I don't think you need `schemeKey` here

Reply:
I had to because `colorSchemes` was an object, so I needed to iterate over `key, value` pairs. Removing `schemeKey` (even though it's not used in this case) would make it only iterate over the keys.

However, since I'm not using those keys anymore, I've now change the JS that generates the data to output an array instead and therefore I can do what you ask :-)